packages/analytics-ma-browser/src/version.ts

```diff
@@ -0,0 +1,2 @@
+/* istanbul ignore next */
```

Comment:
this file will be auto generated on each release and we'll lose this istanbul comment. safer to add to `jest.config.js`

Reply:
however, i noticed that this is not being used. this is being imported in other packages in context plugin. this said, we package will reuse the context plugin of browser and send library info for browser package

Reply:
So should we remove this file?

Reply:
The current state does not use this, but i don't know if we should create a separate context for this one. Check if you think think it's valuable to see a different library value for this package or not. Those values can be used for debugging or adoption metrics

Reply:
Okay, just keep it for now.